examples/circuitplayground_touch_all.py

```diff
@@ -2,20 +2,34 @@
 # SPDX-License-Identifier: MIT
 
 """This example prints to the serial console when you touch the capacitive touch pads."""
+import time
+import board
 from adafruit_circuitplayground import cp
 
+
+# You'll need to first use the touchpads individually to register them as active touchpads
+# You don't have to use the result though
+is_a1_touched = cp.touch_A1  # This result can be used if you want
+if is_a1_touched:
+    print("A1 was touched upon startup!")
+is_a2_touched = cp.touch_A2
+is_a3_touched = cp.touch_A3
+is_a4_touched = cp.touch_A4
+
+print("Pads that are currently setup as touchpads:")
+print(cp.touch_pins)
+
 while True:
-    if cp.touch_A1:
-        print("Touched pad A1")
-    if cp.touch_A2:
-        print("Touched pad A2")
-    if cp.touch_A3:
-        print("Touched pad A3")
-    if cp.touch_A4:
-        print("Touched pad A4")
-    if cp.touch_A5:
-        print("Touched pad A5")
-    if cp.touch_A6:
-        print("Touched pad A6")
-    if cp.touch_TX:
-        print("Touched pad TX")
+
+    current_touched = cp.touched
+
+    if current_touched:
+        print("Touchpads currently registering a touch:")
+        print(current_touched)
+    else:
+        print("No touchpads are currently regustering a touch.")
```

Comment:
Going back to test this again. noticed small typo here, `regustering` instead of `registering`

Reply:
Ha!  Good catch.